forms-flow-web/src/components/Form/Item/View.js

```diff
@@ -196,6 +196,11 @@ const View = React.memo((props) => {
     }
   };
 
+  useEffect(() => {
+    if (form._id && !error) setIsValidResource(true);
+    return () => setIsValidResource(false);
+  }, [error, form._id]);
```

Comment:
Chcek for types of possible errors here. Form validation errors can be ignored for draft

Reply:
We only expect this to run for the first time, so the error that we are concerned here would be `Resource not found` error.

Reply:
Yes . True. The submission and related validation errors will be at submission error api.